src/pages/v4/[chainName]/p/[projectId]/index.tsx

```diff
@@ -53,9 +54,11 @@ const Providers: React.FC<
             metadata: { ipfsGatewayHostname: OPEN_IPFS_GATEWAY_HOSTNAME },
           }}
         >
-          <V4ProjectMetadataProvider projectId={BigInt(projectId)}>
-            {children}
-          </V4ProjectMetadataProvider>
+          <RulesetCountdownProvider>
```

Comment:
Does the whole page need to be aware of this context? Could it just be scoped to wherever the countdown is?

Reply:
Good call, moved it